src/components/RateAgentModal/index.tsx

```diff
@@ -83,10 +91,27 @@ export default function RateAgentModal({
         <RateAgentRatingRadio control={control} name="availability" />
         <RateAgentTags control={control} />
         <div className="mb-5">
-          <h4>
+          <h4>{t('currentlyOnParole')}</h4>
+          <FormCheck
+            type="radio"
+            label="No"
+            name="currentlyOnParole"
+            onChange={() => onClick(false)}
+            required
+          ></FormCheck>
+          <FormCheck
+            type="radio"
+            label="Yes"
+            name="currentlyOnParole"
+            onChange={() => onClick(true)}
+            required
+          ></FormCheck>
+        </div>
+        <div className="mb-5">
```

Comment:
`mb-4` to match the spacing of the rest of the form. 

[Informational] Some day we should probably remove the individual margin-bottoms on all these individual components and just add the `vertical-rhythm` class to the form (a helper that adds mb-4 to all direct children).